client/app/vps/upgrade/vps-upgrade.html

```diff
@@ -1,107 +1,100 @@
-                    </div>
-                    <div class="mt-4 font-weight-bold"
-                         data-translate="vps_configuration_upgradevps_step2_explication3"></div>
-                </cui-wizard-step>
-            </cui-wizard-form>
-        </div>
-    </div>
+                        </span>
+                        <span class="oui-radio__section text-right">
+                            <strong class="d-block">
+                                {{ upgrade.monthPrice }} {{ 'vps_configuration_upgradevps_step1_table_price_monthly' | translate }}
+                            </strong>
+                        </span>
+                    </oui-radio>
+                </div>
+            </div>
+        </oui-step-form>
+        <oui-step-form data-header="{{:: 'vps_configuration_upgradevps_step2_title' | translate }}"
+            data-loading="$ctrl.loaders.step2"
+            data-on-focus="$ctrl.initVpsConditions()">
+            <div data-ng-repeat="contract in $ctrl.order.contracts">
+                <a target="_blank"
+                   class="oui-link oui-link_icon"
+                   data-ng-href="{{ contract.url }}">
+                    <span data-ng-bind="contract.name"></span>
+                    <span class="oui-icon oui-icon-external_link" aria-hidden="true"></span>
+                </a>
+            </div>
+            <oui-checkbox
+                data-text="{{ 'contracts_agree' | translate }}"
+                id="contractsValidated"
+                name="contractsValidated"
+                data-model="$ctrl.conditionsAgree"
+                required>
+            </oui-checkbox>
+        </oui-step-form>
+        <oui-step-form data-header="{{:: 'vps_configuration_upgradevps_step3_title' | translate }}"
+            data-submit-text="{{:: 'vps_configuration_upgradevps_order' | translate }}">
+            <dl class="vps-upgrade-dl-horizontal">
+                <dt class="vps-upgrade-item-horizontal"
+                    data-translate="vps_configuration_upgradevps_step2_price1"
+                    data-translate-values="{ duration: $ctrl.selectedModelForUpgrade.duration.dateFormatted}"></dt>
+                <dd class="vps-upgrade-item-horizontal">
+                    <span data-ng-bind="$ctrl.order.prices.withoutTax.text"></span>
+                    <span data-translate="vps_configuration_upgradevps_step1_table_price_monthly"></span>
+                </dd>
+                <dt class="vps-upgrade-item-horizontal"
+                    data-translate="vps_configuration_upgradevps_step2_price2"></dt>
+                <dd class="vps-upgrade-item-horizontal">
+                    <span data-ng-bind="$ctrl.selectedModelForUpgrade.monthPrice"></span>
+                    <span data-translate="vps_configuration_upgradevps_step1_table_price_monthly"></span>
+                </dd>
+            </dl>
+            <div class="mb-3">
```

Comment:
Does this margin will be always required for each integration? If yes, it should be included in the component I think